tests/pom/agileTravel.page.ts

```diff
@@ -0,0 +1,51 @@
+import { Page } from '@playwright/test';
+
+export class AgileTravelPage {
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async gotoLoginPage() {
+    await this.page.goto('https://travel.agileway.net/login');
+  }
+
+  async login(username: string, password: string) {
+    await this.page.locator('#username').fill(username);
```

Comment:
For better maintainability and readability, it would be great to define the locators (#username, #password, etc.) separately at the start of the function or class. This way, the locators are reusable and easier to update if the selectors change in the future.